bears/vcs/git/GitLinearCommitBear.py

```diff
@@ -0,0 +1,32 @@
+from bears.vcs.VCSCommitMetadataBear import COMMIT_TYPE
+from bears.vcs.git.GitCommitMetadataBear import GitCommitMetadataBear
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+
+
+class GitLinearCommitBear(GlobalBear):
+    LANGUAGES = {'Git'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    BEAR_DEPS = {GitCommitMetadataBear}
+
+    def run(self, dependency_results,
+            git_rebase_help_url: str =
+            'http://www.bitsnbites.eu/a-tidy-linear-git-history/',
+            **kwargs):
+        """
+        Inspect the HEAD commit to check if it is a
+        git merge commit and return result accordingly.
+
+        :param allow_git_merge_commit: Whether merge commits are
```

Comment:
remove parameter.
"GitLinearCommitBear" never allows merge commits, because then it isnt a linear history.

update docstrings and everything else accordingly.